README.md

```diff
@@ -27,26 +29,48 @@ documented below.
 
 ## Include the library
 
-    const InomialClient = require("./InomialClient.js");
+  ```js
```

Comment:
please put this back to use the MD indenting style instead of the backtick style. It's way more readable if you're using a text editor, which is the reason I use the indent style.

Reply:
The style I'm using allows GitHub to do colour syntax highlighting (whereas in yours it doesn't). Both styles are standard markdown. Are you sure you don't want the syntax highlighting?
(I'll roll these changes back anyway, please indicate if you want syntax highlighting otherwise.)